### DISCUSSION TITLE
Is there a way to change displacy's colors in the evaluate output?

### DISCUSSION THREAD

**Discussion by vmatter** [explosion/spaCy#10834] [Help: Coding & Implementations] [answered]
Using the `evaluate` function, I can save displacy's HTML to a chosen directory. However, I am not able to use a similar structure as the usual way to change it:

```
colors = {'Name': "#A4DD00"}

options = {"ents": ['Name'], 
           "colors": colors}
```

Is there a way to change the colors for the `evaluate` function?

Many thanks.

**Comment by ljvmiranda921** [comment, marked as answer]
Hi @vmatter ,

In this case you need to write a custom script. You can use the [`spacy/cli/evaluate.py`](https://github.com/explosion/spaCy/blob/master/spacy/cli/evaluate.py) file for reference. The relevant part here is the [`render_parses`](https://github.com/explosion/spaCy/blob/6be09bbd07243b546ed52d5a03cdf54c8e028566/spacy/cli/evaluate.py#L175) function that calls `displacy.render` (where you can pass your options). Aside from that, it should be the same everywhere else.

**Reply by vmatter** [reply]
Many thanks, I will read the links and implement the custom script.
